apisix/plugins/dynamic-upstream.lua

```diff
@@ -71,7 +71,32 @@ local option_def = {
         },
         minItems = 0,
         maxItems = 10
-    }
+    },
+    {
+        type = "array",
+        items = {
+            {
+                type = "string",
```

Comment:
I think we can support more data type, eg: `number`.

```json
["server_port", "in", [1982, 1983, 1984]]
```

we can implement this feature by new PR.

Reply:
remember to create new issue

Reply:
I will create later.

Reply:
alreadly created.